mxcube3/routes/qutils.py

```diff
@@ -1752,3 +1752,9 @@ def get_task_progress(node, pdata):
 def is_interleaved(node):
     return hasattr(node, "interleave_num_images") and \
         node.interleave_num_images > 0
+
+
+def reset_queue_settings():
```

Comment:
This could be called init, like this you can use it in both __init__ and Login

Reply:
gonna rename